### ISSUE TITLE
Please add Dark mode or Night Mode

### ISSUE THREAD

**Issue by MoeJoe87** [mobnetic/BitcoinChecker#878]
Would be nice to have...cause my android is in Night mode and the only notifications which are still white is from Bitcoin Checker.

**Comment by zbahadir** [comment]
Night mode is very necessary 👍

**Comment by Spitfire75** [comment]
Agreed. A setting under Notification Settings where I could customize the color of the notification too.
